spec/dummy/app/models/task.rb

```diff
@@ -1,6 +1,7 @@
 class Task < ActiveRecord::Base
   has_event :complete
   has_events :archive, skip_scopes: true
+  has_events :expire, strategy: :time_comparison
```

Comment:
Since we test `has_events` here (the plural variant), we can change ` has_events` to  `has_event` (singular) one line above.